src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -128,6 +129,15 @@
     public static final String OUTPUT_DENOISED_COPY_RATIOS_LONG_NAME = "output-denoised-copy-ratios";
     public static final String AUTOSOMAL_REF_COPY_NUMBER_LONG_NAME = "autosomal-ref-copy-number";
     public static final String ALLOSOMAL_CONTIG_LONG_NAME = "allosomal-contig";
+    public static final String INPUT_INTERVALS_LONG_NAME = "input-intervals-vcf";
+    public static final String INPUT_DENOISED_COPY_RATIO_LONG_NAME = "input-denoised-copy-ratio";
```

Comment:
Looks like this isn't used yet, but if it's something you'll add, perhaps change the argument name to `input-denoised-copy- ratios` to be consistent with `output-denoised-copy-ratios`.